core/src/test/scala/tofu/ContextSuite.scala

```diff
@@ -14,9 +14,9 @@ object ContextSuite {
     implicitly[WithContext[ReaderT[F, Ctx, *], Ctx]]
     implicitly[HasLocal[ReaderT[F, Ctx, *], Ctx]]
     implicitly[WithLocal[ReaderT[F, Ctx, *], Ctx]]
-    implicitly[HasProvide[ReaderT[F, Ctx, *], F, Ctx]]
+    implicitly[HasProvide[ReaderT[F, Ctx, *], F, Ctx]](ContextBase.readerTContext[F, Ctx])
```

Comment:
is explicit instance still needed even after aliasing HasProvide to WithProvide?

Reply:
oh, this is garbage, sorry